onecodex/analyses.py

```diff
@@ -177,6 +177,9 @@ def to_df(self, rank='auto', top_n=None, threshold=None, remove_zeros=True, norm
         table_format : {'long', 'wide'}
             If wide, rows are classifications, cols are taxa, elements are counts. If long, rows are
             observations with three cols each: classification_id, tax_id, and count.
+        include_not_top_n : { True, False }
```

Comment:
Did we end up not  using this kwarg?

Reply:
I was going to add this feature but decide that I should not include it in this PR. Thanks for catching.